onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-12.395-12.396.sql

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2016-2017 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * This stored Procedure is part of the eIACUC ETL process and is being added her to create the Stored Procedures in the
+ * SQL 2014 DB for Labkey-gj
+ */
+
+/****** Object:  StoredProcedure [onprc_ehr].[etl1_eIACUCtoPRIMEProcessing]    Script Date: 2/7/2018 2:01:46 PM ******/
+SET ANSI_NULLS ON
```

Comment:
We should really strip out all of these SET ANSI_NULLS ON and SET QUOTED_IDENTIFIER ON statements out of these upgrade scripts. I won't comment on each one individually.

Reply:
done